lib/capybara/rack_test/driver.rb

```diff
@@ -50,8 +50,8 @@ def refresh
     browser.refresh
   end
 
-  def submit(method, path, attributes)
-    browser.submit(method, path, attributes)
+  def submit(method, path, attributes, multipart: false)
+    browser.submit(method, path, attributes, multipart: multipart)
   end
```

Comment:
This was the easiest way to communicate the `multipart` knowledge from the form processing to the browser instance, however this interface change would be breaking if there are any callers which rely on implicit Hash arguments:

```rb
def submit(method, path, attributes, multipart: false)
  # ...
end

submit("GET", "/", "foo" => "bar")   # ! ArgumentError: wrong number of arguments (given 2, expected 3)
submit("GET", "/", {"foo" => "bar"}) # OK
submit("GET", "/", params_hash)      # OK
```

However, the headers sent for a form submission by any browser ultimately _do_ depend on the form value, so I am not sure what would be a superior way to do this.

Perhaps implementing a `submit_multipart()` instead?